kong/pdk/private/checks.lua

```diff
@@ -51,11 +51,19 @@ function checks.validate_header(name, value)
 
   local tvalue = type(value)
   if tvalue ~= "string" then
-    if tvalue == "number" or tvalue == "boolean" then
+    if tvalue == "table" then
+      for _, vv in ipairs(value) do
+        local tvv = type(vv)
+        if tvv ~= "string" then
```

Comment:
If Values inside array are number/boolean type they should also be transformed into string IMO?

Reply:
it's not a big problem, we force the customer to use string value, followed by https://github.com/Kong/kong/blob/master/kong/pdk/private/checks.lua#L84